### ISSUE TITLE
FOUC and static-files-loader question

### ISSUE THREAD

**Issue by jswxwxf** [angular-translate/angular-translate#535]
I have below code that configures the translateProvider.
```js
  $translateProvider.translations('en', {
    
    'login': {
      'email': 'Email Address',
      'password': 'Password',
      'auth_provider': 'Select an Authentication Provider'
    }
  
  });
  
  $translateProvider.useStaticFilesLoader({
    prefix: '/scripts/lang/locale-',
    suffix: '.json'
  })
  
  $translateProvider.preferredLanguage('en').fallbackLanguage('en');
```
I've put the 'en' language in the code to prevent the FOUC, following the angular-translate doc, but why angular-translate still try to load the /scripts/lang/locale-en.json?

I don't want to create another locale-us.json file and put all of the en translations again, it's duplicated.

**Comment by marcalj** [comment]
Any update? :)

**Comment by knalli** [comment]
This issue will be resolved in 2.7.0
